source/jobs/worker_or_atools.rst

```diff
@@ -0,0 +1,176 @@
+How can I run many similar computations conveniently?
+=====================================================
+
+It is often necessary to run the same application on many input files,
+or with many different parameter values.  You can of course manage such
+jobs by hand, or write scripts to do that for you.  However, this is a
+very common scenario, so we developed software to do that for you.
+
+Two general purpose software packages are available:
+
+- the `worker framework <https://github.com/gjbex/worker>`_ (:ref:`worker quickstart <worker framework>` and `worker documentation`_) and
+- `atools <https://github.com/gjbex/atools>`_ (`atools documentation`_).
+
+Both are designed to handle this use case, but with distinct twists.
```

Comment:
maybe replace with this?
Both are designed to handle this use case, but each has its own strengths and weaknesses.